### ISSUE TITLE
Inquiry the training loss

### ISSUE THREAD

**Issue by deadlykitten4** [ZHZisZZ/dllm-trainer#24]
Thanks for your great work!

I have some questions about the loss during training. The loss of the first epoch is very low. Is this common? Or does it depend on the type of task in the dataset being fine-tuned? I am finetuning on code task.

<img width="843" height="258" alt="Image" src="https://github.com/user-attachments/assets/0bb52c4e-e5f2-4ff5-ae41-5cc6f79819de" />

**Comment by ZHZisZZ** [comment]
Hi, could you share the dataset details, code snippet, and launch commands?

**Comment by ZHZisZZ** [comment]
`--model_name_or_path "/model/LLaDA-8B-Instruct"`: If you start from a model that has already been trained on code SFT data (e.g., LLaDA-8B-Instruct), it’s normal for training to begin with a very low loss.

**Comment by ZHZisZZ** [comment]
I’ll close the issue for now, but feel free to reopen it or create a new one if you have any further questions.
